pkgs/misc/emulators/yuzu-mainline/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, mkDerivation, fetchFromGitHub
+, cmake, libressl, SDL2, qtbase, python2, alsaLib
+, useVulkan ? true, vulkan-loader, vulkan-headers
+}:
+
+mkDerivation rec {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  src = fetchFromGitHub {
+    owner = "yuzu-emu";
+    repo = "${pname}";
```

`repo = pname;`

You only need to use `${pname}` if adding a variable to a string.

---

I would use something like

```suggestion
    repo = "yuzu-mainline"; # they use a separate repo for mainline “branch”
```

Not really a fan of re-using `pname` in `repo` since it does not often match the project name.

---

I think `repo = pname;` makes sense here, since if you were reusing this package to build yuzu-nightly or master you wouldn't have to replace this string manually. Also makes writing package overrides a tad simpler.